reflex/.templates/apps/base/code/base.py

```diff
@@ -0,0 +1,72 @@
+"""Welcome to Reflex! This file outlines the steps to create a basic app."""
+import reflex as rx
+
+from .sidebar import sidebar
+from .styles import *
+
+
+def content():
+    return rx.box(
+        rx.vstack(
+            rx.heading(
+                "Dashboard",
+                size="3em",
+            ),
+            rx.text(
+                "Welcome to Reflex!",
+            ),
+            rx.text(
+                "You can use this template to get started with Reflex.",
+            ),
+            width="100%",
+            align_items="flex-start",
+            height="90%",
+            box_shadow="0px 0px 0px 1px rgba(84, 82, 95, 0.14)",
+            border_radius=border_radius,
+            padding="1em",
+        ),
+        height="100vh",
+        width="100%",
+        padding_top="5em",
+        padding_x="2em",
+    )
+
+
+def menu_button():
+    return rx.box(
+        rx.menu(
+            rx.menu_button(
+                rx.icon(
+                    tag="hamburger",
+                    size="4em",
+                    color=text_color,
+                ),
+            ),
+            rx.menu_list(
+                rx.menu_item("Home"),
+                rx.menu_divider(),
+                rx.menu_item("About"),
+                rx.menu_item("Contact"),
```

Comment:
how to make these actually work? 

```python
                rx.menu_item("About", on_click=rx.redirect("/about")),
                rx.menu_item(rx.link("Contact", href="/contact")),
```

The `rx.link` one only works when you click on the text itself, but the `on_click` one seems a bit unusual too. Need example so users know what to do.

Reply:
Added a few different link example, setting width to 100% fixed the only working when clicking text